src/components/spreadsheet-view/README.md

```
@@ -0,0 +1,368 @@
+
+-----
+
+## Node Aliases
+
+### Purpose
+
+Node aliases allow displaying side-by-side data from **multiple nodes** of the modification tree in the same spreadsheet. This is used for scenario comparison: for example, comparing the current node's results against a reference node.
+
+### Data model
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)